include/fmt/core.h

```diff
@@ -187,6 +187,12 @@
 #  define FMT_FALLTHROUGH
 #endif
 
+#if FMT_HAS_CPP17_ATTRIBUTE(nodiscard)
+#  define FMT_NODISCARD [[nodiscard]]
+#else
+#  define FMT_NODISCARD
+#endif
```

Comment:
Let's wrap this in `#ifndef FMT_NODISCARD` to allow users override autodetection.

Reply:
Ack

Reply:
Please apply clang-format to your changes.

Reply:
Sorry about that; done.

(Would it be worthwhile to have an automated checker for this?)

Reply:
> Would it be worthwhile to have an automated checker for this?

Sure, a PR is welcome =)

Reply:
> Sure, a PR is welcome =)

Challenge accepted (https://github.com/fmtlib/fmt/pull/2613), but the tools are somewhat crude. Is there a time budget for the CI ?

In another project we grep the output of running git-clang-format, but perhaps I should just contribute/port --dry-run from clang-format to git-clang-format.